landoapi/app.py

```diff
@@ -46,6 +59,40 @@ def create_app(version_path):
     return app
 
 
+def initialize_sentry(flask_app, release, environment):
+    """Initialize Sentry application monitoring.
+
+    See https://docs.sentry.io/clients/python/advanced/#client-arguments for
+    details about what this function's arguments mean to Sentry.
+
+    Args:
+        flask_app: A Flask() instance.
+        release: A string representing this application release number (such as
+            a git sha).  Will be used as the Sentry "release" identifier. See
+            the Sentry client configuration docs for details.
+        environment: A string representing the application environment, such
+            as 'dev', 'stage', or 'prod'.  Will be used as the Sentry
+            "environment" identifier.  See the Sentry client configuration docs
+            for details.
+    """
+    sentry_dsn = os.environ.get('SENTRY_DSN', None)
```

Comment:
The environment variable 'SENTRY_DSN' should be present in our docker-compose file even if it is empty or just a comment because our docker-compose defines all the environment variables needed to run the application. It is not used in production of course, it is just a good way to remind us what will be in the environment.